spec/features/search_page_spec.rb

```diff
@@ -27,6 +27,16 @@
     expect(page).to have_css('a', text: dataset_title)
   end
 
+  scenario 'Search results indicate dataset availability' do
+    index(DatasetBuilder.new.build)
```

Comment:
It might be safer to explicitly set `.with_released(true)` here so we're not relying on the default behaviour of `DatasetBuilder`.